### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +14,15 @@ For more information please read the [documentation][docs].
 
 ## Installation
 
+To install Still you add it to your dependency list. You should be able to
+add it to any mix project.
+
 ### For new projects
 
-`mix archive.install hex still` to install it on your system. You only need to do this once.
+Run `mix archive.install hex still` to install the archive on your system.
```
Comment (frm) [inline README.md:22]: It's not `archive.install`, for now, is it?

Comment (gabrielpoca) [reply README.md:22]: It is, but not from hex. I'll update after I merge the other PR.

Comment (frm) [reply README.md:22]: This is now correct, right? We can archive install it.

Comment (gabrielpoca) [reply README.md:22]: No, you can archive install it from github, not from hex.

**Review by frm — APPROVED** [review]

**Review by frm — APPROVED** [review]

**Pull request merged**